tests/integration/query.ts

```diff
@@ -134,11 +118,13 @@ describe('ensure correct results from latest request', () => {
     expect(answers.state.query.input).toEqual(queries[2]);
     expect(answers.state.vertical.results).toEqual([queries[2]]);
     expect(updateResult.mock.calls).toHaveLength(2);
+    removeListener();
```

Comment:
I think we shouldn't need these remove listener calls anymore?

Reply:
I mean, we don't remove listeners when somebody closes their webpage right? I feel like having code that isn't needed is confusing to people who look at the tests in the future and have to wonder why this listener is being removed. It might lead to confusion like "Are they removing the listener because the headless instance is being shared?".

Reply:
That makes sense, updated!